apps/test-pololu370-encoder/src/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Logger.h"
+#include "Pololu37D.h"
+
+void read_all();
+
+// first pin is yellow wire
+// second pin is white wire
+constexpr Encoder::Pololu37D::Config config = {PA_5, PA_6, NC, 0, GPIO::QEI::Encoding::X4_ENCODING};
```

Comment:
Since this driver will only be using X4 encoding, this shouldn't be configurable by the user. Remove `quadratureEncoding` from the config struct and directly use `X4_ENCODING` inside the constructor

Reply:
Makes sense, I've changed it so the user has no control over the type of encoding used. Also, when it builds the QEI object it always does so with X4 encoding.